src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  line-height: 300px;
+  min-width: 300px;
+  color: white;
+  font-size: 100px;
+  text-align: center;
+  flex-grow: 1;
+}
+
+.block--1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .block--3 {
+    order: 1 ;
+  }
+
+  .block--5,
+  .block--6 {
+    order: 2;
+  }
```

Comment:
? two blocks have the same order?

Reply:
Did you mean #3 of Checklist? I'll correct it.